use std::error::Error;

#[tokio::main]
async fn main() {
    println!("Hello, world!");
}
